src/DacpacTool/PackageBuilder.cs

```diff
@@ -112,19 +112,26 @@ public bool ValidateModel()
                 if (modelError.Severity == ModelErrorSeverity.Error)
                 {
                     validationErrors++;
+                    Console.WriteLine(modelError.GetOutputMessage(modelError.Severity));
                 }
-                else if (modelError.Severity == ModelErrorSeverity.Warning && TreatTSqlWarningsAsErrors)
+                else if (modelError.Severity == ModelErrorSeverity.Warning)
                 {
                     if (!_suppressedWarnings.Contains(modelError.ErrorCode))
```

Comment:
This is getting a little messy with multiple nested if's. Perhaps we could fold the first two if's here into a single method call.

Reply:
To `if..else` statement we can switch to `switch` statement, it will reduce if'ing on top level.